fhir-persistence-jdbc/src/test/java/com/ibm/fhir/persistence/jdbc/test/spec/R4JDBCExamplesTest.java

```diff
@@ -40,16 +38,13 @@ public R4JDBCExamplesTest() throws Exception {
         this.properties = TestUtil.readTestProperties("test.jdbc.properties");
     }
 
-    @BeforeSuite
-    public void bootstrapAndLoad() {
-        System.out.println("Bootstrapping database:");
-
-
-        System.out.println("Processing examples:");
-    }
-
-    @Test(groups = { "jdbc-seed" })
+    @Test(groups = { "jdbc-seed" }, singleThreaded = true, priority = -1)
     public void perform() throws Exception {
+        if(this.database == null) {
+            System.out.println("Bootstrapping database:");
+            this.database = new DerbyFhirDatabase(DerbyInitializer.DB_NAME);
+        }
```

Comment:
just curious, maybe long time ago, these codes were in bootstrapAndLoad() function?

Reply:
no - I added, and move the output statement.